api4/system.go

```diff
@@ -545,3 +548,70 @@ func getServerBusyExpires(c *Context, w http.ResponseWriter, r *http.Request) {
 	}
 	w.Write([]byte(c.App.Srv().Busy.ToJson()))
 }
+
+func upgradeToEnterprise(c *Context, w http.ResponseWriter, r *http.Request) {
+	if !c.App.SessionHasPermissionTo(*c.App.Session(), model.PERMISSION_MANAGE_SYSTEM) {
+		c.SetPermissionError(model.PERMISSION_MANAGE_SYSTEM)
+		return
+	}
+
+	auditRec := c.MakeAuditRecord("upgradeToEnterprise", audit.Fail)
+	defer c.LogAuditRec(auditRec)
+
+	percentage, _ := c.App.Srv().UpgradeToE0Status()
```

Comment:
maybe call the `UpgradeToE0Status()` after checking for `BuildEnterpriseReady == true` ?

Reply:
Done